test/functional/forall/atomic-ref/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+raja_add_test(
+  NAME test-forall-atomic-ref-math-seq
+  SOURCES test-forall-atomic-ref-math-seq.cpp)
+
+raja_add_test(
+  NAME test-forall-atomic-ref-other-seq
+  SOURCES test-forall-atomic-ref-other-seq.cpp)
+
+if(RAJA_ENABLE_OPENMP)
+  raja_add_test(
+    NAME test-forall-atomic-ref-math-openmp
+    SOURCES test-forall-atomic-ref-math-openmp.cpp)
+
+  raja_add_test(
+    NAME test-forall-atomic-ref-other-openmp
+    SOURCES test-forall-atomic-ref-other-openmp.cpp)
+endif()
+
+if(RAJA_ENABLE_TARGET_OPENMP)
```

Comment:
OpenMP target and TBB are not included because we have no atomic policies for those?

Reply:
There were no previous test cases for these. Apparently TBB atomics are deprecated https://software.intel.com/en-us/node/506277. I will trying reusing the existing tests with OMP Target loops.

Reply:
That's a good idea. You should be able to use regular OpenMP atomic policies.